app.js

```diff
@@ -1,75 +1,91 @@
 
 //Greeting
 var userName = prompt ('What\'s your name?');
-if(userName == null || userName === ""){
-  userName = "guest";
-}
+// if(userName == null || userName === ''){
+//   userName = 'guest';
+// }
+if (!userName) userName = 'guest';
 alert('Hello ' + userName + '. Thanks for coming to my site!');
 
-//Setup Arrays
-var questionArray = ['Did I attend UC Berkeley?', 'Is fullstack development part of my long term goals?', 'Do I currently work as a SDET?', 'Is Paris my dream vacation destination?', 'Do I have a dog named, Eli?', 'What is one of my favorite dishes?'];
-var myAnswerArray = ['yes', 'yes', 'yes', 'yes'];
-var inputAnswerArray = ['yes', 'y', 'beef noodle soup', 'prime rib'];
-
+//Setup //8 questions
+var questionArray = ['Did I attend UC Berkeley?', 'Is fullstack development part of my long term goals?', 'Do I currently work as a SDET?', 'Is Paris my dream vacation destination?', 'Do I have a dog named, Eli?', 'What is one of my favorite dishes?', 'Guess a number between 1 to 100', 'What are my favorite food destination cities?'];
```

Comment:
Glad you added the question arrays and answer arrays back in after I goofed and gave you bad info. This is a fantastic way of organizing this assignment.